utils/motorcontroller/SparkMAXFactory.java

```diff
@@ -38,13 +42,33 @@ public static class Config {
 
     }
 
-    public static final Config defaultConfig = new Config();
-    public static final Config defaultFollowerConfig = new Config();
+    public static final Config defaultConfig = new Config(); // For motors where we care about position and velocity
+    public static final Config defaultVelocityConfig = new Config();// For motors were we only care about precise velocity and not precise position 
+    public static final Config defaultPositionConfig = new Config(); // For motors were we only care about precise position and not precise velocity 
+    public static final Config defaultLowUpdateRateConfig = new Config(); // For motors where we neither care about precise velocity or position
+    public static final Config defaultFollowerConfig = new Config(); //For follower motors
 
     static {
         defaultFollowerConfig.STATUS_FRAME_0_RATE_MS = 1000;
```

Comment:
we could arguably set the follower frames all to 65535 (the max). As a more conservative intermediate change, let's set them all to the same 1000. I can't think of a reason we'd want 3 and 4 more frequently and 5 and 6 less frequently. We're not using any of 3-6, at least this year